dcc-portal-ui/app/scripts/analysis/js/analysis.js

```diff
@@ -200,6 +212,34 @@
 
   var module = angular.module('icgc.analysis.services', ['restangular']);
 
+  var analysesStrings = {
+      set: {
+        name: 'Set Operations',
+        description: 'Display Venn diagram and find out intersection or union, etc. of your sets of the same type.',
+        demoDescription: 'Compare high impact mutations in brain cancers across GBM-US, LGG-US, and PCBA-DE.'
+      },
+      get union () {
+        return this.set;
+      },
+      enrichment: {
+        name: 'Enrichment Analysis',
+        description: 'Find out statistically significantly over-represented groups of gene sets ' +
+        '(e.g. Reactome pathways) when comparing with your gene set.',
+        demoDescription: 'Perform enrichment analysis on top 50 genes in Cancer Gene Census.'
+      },
+      phenotype: {
+        name: 'Phenotype Comparison',
+        description: 'Compare some characteristics (e.g. gender, vital status and age at diagnosis)' +
+        ' between your donor sets.',
+        demoDescription: 'Compare phenotypes across brain, breast, and colorectal cancer donors.'
+      },
+      survival: {
+        name: 'Survival Analysis',
+        description: 'TBD',
```

Comment:
This section _might_ get removed as I don't know if was intended to be a standalone experiment type